src/site/paragraphs/lists_of_links.drupal.liquid

```diff
@@ -0,0 +1,30 @@
+<!-- Section header -->
+<h2>{{ entity.fieldSectionHeader }}</h2>
+
+<div class="vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-u-flex-direction--row medium-screen:vads-u-flex-wrap--wrap">
+  {% for vaParagraph in entity.fieldVaParagraphs %}
+    {% assign hasEvenIndex = forloop.index | modulo: 2 %}
+    <div class="vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-l-col--5{% if hasEvenIndex %} medium-screen:vads-u-margin-right--8{% endif %}">
+      <!-- Section header -->
+      <h3>{{ vaParagraph.entity.fieldSectionHeader }}</h3>
+
+      <ul class="usa-unstyled-list">
+        <!-- Links -->
+        {% for fieldLink in vaParagraph.entity.fieldLinks %}
+          <li class="vads-u-padding-y--1">
+            <a href="{{ fieldLink.url.path }}" rel="noreferrer noopener">
```

Comment:
I don't think you need ` rel="noreferrer noopener"` bc all of the links will be just to other pages in the LC